api/src/config/sequelize.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  development: {
+    username: process.env.DEV_DB_USERNAME,
+    password: process.env.DEV_DB_PASSWORD,
+    database: process.env.DEV_DB_NAME,
+    host: process.env.DEV_DB_HOSTNAME,
+    dialect: 'postgres',
+    operatorsAliases: false,
+  },
+  test: {
+    username: 'postgres',
+    password: 'qweqwe',
+    database: 'grow_roadmaps',
+    host: 'localhost',
+    dialect: 'postgres',
+    operatorsAliases: false,
+    logging: false,
+  },
+  production: {
+    username: process.env.PROD_DB_USERNAME,
```

Comment:
Para mim não precisamos de `process.env.PROD_DB_USERNAME` e `process.env.DEV_DB_USERNAME`. Apenas uma config `process.env.DB_USERNAME` é o bastante. sequelize.config.js (e a aplicação) não precisa se preocupar em quantos ambientes nós temos e se for o caso, mudamos as variáveis de ambiente diretamente ou via .env .  
Pelo o que posso ver, dá pra passar o caminho absoluto para um .env file, então se alguem quiser rodar dois ambientes (dev e test) é só usar o parametero `dotenv_config_path` . 

Reference: https://12factor.net/confighttps://12factor.net/config

O que acham? Não vejo isso como um blocker, então se curtirem posso mudar em outra pull request.

Reply:
Reference unavailable

Reply:
https://12factor.net/config 😇  Valeu.